test/Shared/MsBuildProjectStrings.cs

```diff
@@ -35,25 +34,24 @@ public static string GetNugetConfigTxt(string artifactsDir)
 
   <ItemGroup>
     <PackageReference Include=""Microsoft.ApplicationInsights.AspNetCore"" Version=""2.0.0-beta1"" />
-    <PackageReference Include=""Microsoft.AspNetCore"" Version=""2.0.0-preview3-*"" />
-    <PackageReference Include=""Microsoft.AspNetCore.Mvc"" Version=""2.0.0-preview3-*"" />
-    <PackageReference Include=""Microsoft.AspNetCore.StaticFiles"" Version=""2.0.0-preview3-*"" />
-    <PackageReference Include=""Microsoft.Extensions.Logging.Debug"" Version=""2.0.0-preview3-*"" />
-    <PackageReference Include=""Microsoft.VisualStudio.Web.BrowserLink"" Version=""2.0.0-preview3-*"" />
+    <PackageReference Include=""Microsoft.AspNetCore"" Version=""{0}"" />
+    <PackageReference Include=""Microsoft.AspNetCore.Mvc"" Version=""{0}"" />
+    <PackageReference Include=""Microsoft.AspNetCore.StaticFiles"" Version=""{0}"" />
+    <PackageReference Include=""Microsoft.Extensions.Logging.Debug"" Version=""{0}"" />
+    <PackageReference Include=""Microsoft.VisualStudio.Web.BrowserLink"" Version=""{0}"" />
   </ItemGroup>
   <ItemGroup>
-    <PackageReference Include=""Microsoft.AspNetCore.Authentication.Cookies"" Version=""2.0.0-preview3-*"" />
-    <PackageReference Include=""Microsoft.AspNetCore.Diagnostics.EntityFrameworkCore"" Version=""2.0.0-preview3-*"" />
-    <PackageReference Include=""Microsoft.AspNetCore.Identity.EntityFrameworkCore"" Version=""2.0.0-preview3-*"" />
-    <PackageReference Include=""Microsoft.EntityFrameworkCore.Design"" Version=""2.0.0-preview3-*"" />
-    <PackageReference Include=""Microsoft.EntityFrameworkCore.SqlServer"" Version=""2.0.0-preview3-*"" />
-    <PackageReference Include=""Microsoft.EntityFrameworkCore.SqlServer.Design"" Version=""2.0.0-preview3-*"" />
-    <PackageReference Include=""Microsoft.Extensions.Configuration.UserSecrets"" Version=""2.0.0-preview3-*"" />
-    <PackageReference Include=""Microsoft.VisualStudio.Web.CodeGeneration.Design"" Version=""2.0.0-preview3-*"" />
-    <PackageReference Include=""Microsoft.VisualStudio.Web.CodeGeneration.Tools"" Version=""2.0.0-preview3-*"" />
+    <PackageReference Include=""Microsoft.AspNetCore.Authentication.Cookies"" Version=""{0}"" />
+    <PackageReference Include=""Microsoft.AspNetCore.Diagnostics.EntityFrameworkCore"" Version=""{0}"" />
+    <PackageReference Include=""Microsoft.AspNetCore.Identity.EntityFrameworkCore"" Version=""{0}"" />
+    <PackageReference Include=""Microsoft.EntityFrameworkCore.Design"" Version=""{0}"" />
+    <PackageReference Include=""Microsoft.EntityFrameworkCore.SqlServer"" Version=""{0}"" />
```

Comment:
Yup, the SqlServer.Design package has been removed by EF https://github.com/aspnet/EntityFramework/commit/32434d2d86a75e091e331940384115e447ff33de